ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/activityBlock/activityBlock.component.ts

```diff
@@ -29,12 +29,14 @@ import { ModalActivityBlockComponent } from '../modalActivityBlock/modalActivity
 })
 export class ActivityBlockComponent implements OnInit, OnDestroy {
     @Input() block: ActivityActivityBlock;
+    @Input() enabled: boolean;
```

Comment:
we introduced new input "enabled" in multiple components.But is it relly neccesary? Maybe we can reuse "readonly" input? Because in fact they are synonims and it's confusing when look at both of them about what's the responcibility area of each of them. My suggestions either use readonly input, either rename enabled input to smth more specific, either add a comment to distinguish between then

Reply:
I understand, it can be confusing at first. The main difference between `enabled` and `readonly` properties is that `enabled` is reevaluated each time we make a `PATCH` request (just like with block visibility).

Reply:
I see, that's clear now, thanks!